src/app/features/auth/auth.service.ts

```diff
@@ -0,0 +1,36 @@
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { ENDPOINTS } from '@shared/api/endpoints';
+import { CookieService } from 'ngx-cookie-service';
+
+import { environment } from 'src/environment';
+import { AuthResponse, User } from './shared/auth.iterfaces';
+import { AuthActions } from './store/auth.actions';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  private http = inject(HttpClient);
+  private cookieService = inject(CookieService);
+  private store = inject<Store<AuthResponse>>(Store);
+
+  login(email: string, password: string) {
+    return this.http.post<AuthResponse>(`${environment.API_URL}/login`, { email, password });
```

Comment:
docelowy endpoint na login jest zapisany w słowniku `shared>endpoints.ts` (auth/login) - pamiętać, żeby zmienić, gdy przestaniemy korzystać z json-server

Reply:
dodajmy zatem do backlogu taska na to oraz komentarz w kodzie z numerem tego taska z onaczeniem `// TODO WHEN BACKEND READY `